packages/ledger-plugin/fabric/validator/unit-test/fabric-docker/README.md

```diff
@@ -11,9 +11,9 @@
 
 ## a) clone the directories
 	curl -sSL https://bit.ly/2ysbOFE | bash -s -- 1.4.0 1.4.0
-Then, `fabric-samples` directory is made in the current directory, including `basic-network`, `bin`, `chaincode/fabcar`, and `fabcar`. 
+Then, `fabric-samples` directory is made in the current directory, including `basic-network`, `bin`, `chaincode/fabcar`, and `fabcar`.
 
-- c.f.: https://hyperledger-fabric.readthedocs.io/en/master/install.html#install-samples-binaries-and-docker-images
+- c.f.: https://hyperledger-fabric.readthedocs.io/en/release-2.2/install.html
```

Comment:
Would you change the URL to the following URL?
https://hyperledger-fabric.readthedocs.io/en/release-1.4/install.html

Reply:
> Would you change the URL to the following URL?
> https://hyperledger-fabric.readthedocs.io/en/release-1.4/install.html

@takeutak Sure thing! Here's the commit: 2e13877b45a83ad455b6c80a45b370c2b6f509fa